astroML/lumfunc.py

```diff
@@ -179,23 +195,42 @@ def bootstrap_Cminus(x, y, xmax, ymax, xbins, ybins,
     normalize : boolean
         if true, then returned distributions are normalized.  Default
         is False.
+    get_cumulative : boolean
+        if true, then cumulative distributions are returned in addition
+        to the standard distributions. Default is False.
 
     Returns
     -------
     dist_x, err_x, dist_y, err_y : ndarrays
         distributions of size Nbins_x and Nbins_y
+    cuml_x, cuml_y : ndarrays
+        distributions the size Nbins_x and Nbins_y
```

Comment:
Just noticed the order of return values. This should be explicitly documented here